.github/workflows/npm-publish-github-packages.yml

```diff
@@ -4,26 +4,23 @@
 name: Node.js Package
 
 on:
-  workflow_dispatch:
-  release:
-    types: # This configuration does not affect the page_build event above
-      - created
+    release:
+        types: [created]
 
 jobs:
-  publish-npm:
-    runs-on: ubuntu-latest
-    permissions:
-      contents: read
-      packages: write
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
-        with:
-          node-version: 17.2
-          registry-url: https://registry.npmjs.org/
-      - run: npm ci
-      - run: npm run build
-      - run: npm publish
-        env:
-          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
-          NPM_TOKEN: ${{secrets.NPM_TOKEN}}
+    publish:
+        runs-on: ubuntu-latest
+        permissions:
+            contents: read
+            packages: write
+        steps:
+            - uses: actions/checkout@v3
+            - uses: actions/setup-node@v3
+              with:
+                  node-version: 17.2
+                  registry-url: https://npm.pkg.github.com/
+            - run: npm ci
+            - run: npm run build
+            - run: npm publish
+              env:
+                  NODE_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Have you tested this? I had problems with variables named `GITHUB_TOKEN`

Reply:
Haven't tested since this isn't merged and no release has been created. Was going to make a release right after merge to test of course, since I am doing this to be able to use this package in the hls-to-mp4 job. 
Not sure why you've had that issue since this is technically supposed to work, but if we run into the issue again with this one, we have that much more reason to solve that issue lol

Reply:
Looking in the repo settings, there is a spot to put secrets to be used in GH actions, so maybe the issue you've had before with this was because there wasn't a `GITHUB_TOKEN` added in there maybe?

Reply:
Yup, I know about the repo/org/etc secrets, but `GITHUB_TOKEN` is reserved. IIRC anything starting with `GITHUB_` will throw an error when you try to add it as a secret. https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret

Reply:
oh alright, gotcha! I wonder if we should start using `GH_TOKEN` instead then maybe? That way it's still descriptive, but won't interfere with the token they provide in the action

Reply:
On GTC I added a PR check github action and used `GH_NPM_TOKEN`. Whatever we should do, we should be consistent because you'll need to add it to your local shell's environment variables

Reply:
I kinda dislike adding `npm` to the token name since this token has nothing to do with npm, and the package will be hosted in GH.. I agree we should have consistency though